Source/WebCore/loader/FrameLoader.cpp

```diff
@@ -1824,6 +1824,12 @@ void FrameLoader::loadWithDocumentLoader(DocumentLoader* loader, FrameLoadType t
     if (!isNavigationAllowed())
         return;
 
+    if (RefPtr page = frame->page())
+        m_loadStartedDuringSwipeAnimation = page->isInSwipeAnimation();
+    else
+        m_loadStartedDuringSwipeAnimation = false;
```

Comment:
Does this state need to persist after the animations and transitions are done?  I think this state would be better on the DocumentLoader because the main Frame can last many navigations but the DocumentLoader seems to have a more similar scope to the lifetime of this bool.

Reply:
It does not, so have moved the bool to the DocumentLoader!